src/ITensorNetworks.jl

```diff
@@ -40,6 +40,8 @@ include("solvers/local_solvers/contract.jl")
 include("solvers/local_solvers/linsolve.jl")
 include("treetensornetworks/abstracttreetensornetwork.jl")
 include("treetensornetworks/treetensornetwork.jl")
+include("treetensornetworks/matelem.jl")
+include("treetensornetworks/qnarrelem.jl")
 include("treetensornetworks/opsum_to_ttn.jl")
```

Comment:
Probably we should make a `treetensornetworks/opsum_to_ttn` directory where we put these three files to make it clearer they are meant to be used with `opsum_to_ttn.jl`.